Tests/README.md

```diff
@@ -0,0 +1,48 @@
+# Testsuite for SCS standards
+
+The tool `scs-compliance-check.py` parses a 
+[compliance definition file](https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0003-v1-sovereign-cloud-standards-yaml.md)
```

Comment:
we should add a symlink in the repository, that always leads to the latest version, what do you think?